release notes/v0.36.0.md

```diff
@@ -0,0 +1,132 @@
+k6 v0.36.0 is here! 🎉 It introduces a couple of new features which enhance its usability, includes a number of fixes, and the result of ongoing refactoring efforts.
+
+## New Features!
+
+### Sourcemaps support ([#2082](https://github.com/grafana/k6/pull/2082))
```

Comment:
This section will likely need an addition of the fresh fix from yesterday https://github.com/grafana/k6/pull/2345

Reply:
How would you phrase and summarize that? 😸

Reply:
I think just adding a phrase about the file limit will suffice. "Note that in case the file size of input is greater than X byes, generation of source maps is disabled with a warning." Maybe add something about reasoning too, ~ "to avoid OOM after babel transform"?
CC @MStoykov could you please check :slightly_smiling_face: 

Going by some comments in #2345, `K6_DEBUG_SOURCEMAP_FILESIZE_LIMIT` should not be advertised. But in this fix, behavior of a new feature changes depending on input and IMO, users should be warned to keep an eye on that.

Reply:
We can mention `K6_DEBUG_SOURCEMAP_FILESIZE_LIMIT` here, with the same NB that `GODEBUG` change had, that it's a temporary workaround that will probably go away in a future k6 version.

Reply:
It only gets disabled in/for babel, other than that it works. So more accurately

"Note that if a file size is greater than 250kb and the internal babel is needed, babel will not generate source map. This is because during testing it was found this takes 3x-4x more memory leading to OOMs on bigger inputs. A pregenerated sourcemap will always be loaded." 

for some numbers that I don't know if we want to add but `sample/http_2.js` with it's body copied to be 248kb takes 44s and 2270320 bytes of memory or 42s and 967132 bytes of memory, with and without sourcemaps.